src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/CreateAssignmentController.java

```diff
@@ -76,8 +76,7 @@ public CreateAssignmentController(@Autowired CreateRoleAssignmentOrchestrator cr
         roleName = "T(uk.gov.hmcts.reform.roleassignment.util.AuditLoggerUtil).buildRoleNames(#result)",
         caseId = "T(uk.gov.hmcts.reform.roleassignment.util.AuditLoggerUtil).buildCaseIds(#result)",
         assignerId = "#assignmentRequest.request.assignerId",
-        correlationId = "#correlationId",
-        requestPayload = "#auditContextWith.requestPayload"
+        correlationId = "#correlationId"
```

Comment:
what is the reason to removing this payload?

Reply:
I think, we shouldn't remove this from any endpoint...If create assignment call is taking longer it would be good to review its request payload.

Reply:
I added the requestPayload for logging for the endpoints